### PR TITLE
updates the limitation section with latest test values

### PR DESCRIPTION
**Pull request by mcka1n**
### Changes

* I'm updating the Bolt Server (machine) specs that we used on our latest load tests.
* And adjusting the Puppet version from 8 to 7.XX

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,9 +155,9 @@ curl -d "@test_params.json" --insecure --header "$auth_header" "$uri"
 ## Limitations
 
 - `puppet_bolt_server` is tested only on RHEL 7 and 8 based systems.
-- Requires Puppet >= 6.21.0 < 8.0.0
+- Requires Puppet >= 6.21.0 < 7.18.0
```
Comment (timidri) [inline README.md]: The last version of Puppet is 7.20.0 - should we say <= 7.20.0?

**Review by timidri — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mcka1n) [reply README.md]: ha! I was living a couple of versions in the past

Updated!

File: README.md [hunk 2]
```diff
@@ -155,7 +155,7 @@ curl -d "@test_params.json" --insecure --header "$auth_header" "$uri"
 ## Limitations
 
 - `puppet_bolt_server` is tested only on RHEL 7 and 8 based systems.
-- Requires Puppet >= 6.21.0 < 7.18.0
+- Requires Puppet >= 6.21.0 < 7.20.0
```
Comment (timidri) [inline README.md]: Why `< 7.20.0` which excludes `7.20.0` instead of the suggested `<= 7.20.0`?

**Review by timidri — CHANGES_REQUESTED** [review]

**Review by timidri — APPROVED** [review]
LGTM!

**Pull request merged**